src/braille/internal/braille.cpp

```diff
@@ -1823,14 +2407,15 @@ QString Braille::brailleMeasure(Measure* measure, int staffCount)
             resetOctave(staffCount);
 
             // Undo filling the missing beats with rests, so we don't have an altered score.
-            score->undoRedo(true, nullptr);
-            score->undoRedo(true, nullptr);
-            score->deselectAll();
+            m_score->undoRedo(true, nullptr);
+            m_score->undoRedo(true, nullptr);
+            m_score->deselectAll();
```

Comment:
The corresponding code in line +2377 to +2387 is commented out. Shouldn't this be commented out too?

Also, there seems to be some duplication of this code in `Braille::brailleMeasure` and `Braille::brailleMeasureItems`. Can/should that be prevented?